scss/_patterns_accordion.scss

```
@@ -1,5 +1,11 @@
 @import 'settings';
 
+@mixin align-icon($icon-size: 1rem, $line-height: default-text) {
+  // calculate icon top position based on top padding (from %single-border-text-vpadding--scaling)
+  // and offset to align it with baseline of accordion tab text element
+  top: $table-cell-vertical-padding + 0.5 * (map-get($line-heights, $line-height) - $icon-size);
```

It will be a bit safer to wrap it all in `calc` (https://sass-lang.com/documentation/values/calculations) so that SCSS can be smart about it if at any point these variables are not all using the same units.

```suggestion
  top: calc($table-cell-vertical-padding + 0.5 * (map-get($line-heights, $line-height) - $icon-size));
```

---

:+1: